src/Categories/Morphism/Properties.agda

```diff
@@ -150,3 +150,66 @@ _∘↠_ : B ↠ C → A ↠ B → A ↠ C
 f ∘↠ g = record { mor = mor f ∘ mor g ; epi = Epi-∘ (epi f) (epi g) }
   where
     open _↠_
+
+--------------------------------------------------------------------------------
+-- Isomorphism from a section and a retraction
+
+EpicRetract⇒Iso : ∀ {X Y} {f : Y ⇒ X} {r : X ⇒ Y} → 
+                   r RetractOf f → Epi f → Iso f r
+EpicRetract⇒Iso {X} {Y} {f} {r} rf≈id e = record { 
```

Comment:
Perhaps call it `epi` rather than just `e`?

Reply:
I made this change, thanks.